docs/fiddles/communication/two-processes/synchronous-messages/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  </head>
+  <body>
+    <div>
+      <div>
+        <h1>Synchronous messages</h1>
```

Comment:
I'd still include the "supports" message, but not as a H1. For example:
```suggestion
        <h1>Synchronous messages</h1>
        <i>Supports: Win, macOS, Linux <span>|</span> Process: Both</i>
```

Reply:
@jkleinsc  I have pushed the changes requested.